### DISCUSSION TITLE
Missing characters in any Terminal app I use and losing command history output when I reopen app

### DISCUSSION THREAD

**Discussion by filipebcs** [romkatv/powerlevel10k#2662] [General]
Hi, guys.

I have installed Oh My Zsh on my Mac and I have been using the Tabby Terminal  Version 1.0.207 (1.0.207), but this problem happens in other terminal apps as well.

I have configure powerlevel10k as instructed here on its GitHub page and I have some icons which are not being rendered correctly, such as the apple logo pointed out below.

<img width="992" alt="image" src="https://github.com/romkatv/powerlevel10k/assets/11351791/fd49e0d9-0439-4ae5-88c1-c490258e68a7">

Also, I tried to echo the apple logo using its unicode, but it shows me a weird symbol instead.

<img width="991" alt="image" src="https://github.com/romkatv/powerlevel10k/assets/11351791/0128f9ac-fb95-4024-ae8c-e67a01113e45">

I don't know why this is happening since I have MESLOLGS NF font installed in my system as shown below and Tabby is configured to use them

<img width="1112" alt="image" src="https://github.com/romkatv/powerlevel10k/assets/11351791/071d0315-fa48-4966-a0b8-7efae64b10ad">

<img width="1140" alt="image" src="https://github.com/romkatv/powerlevel10k/assets/11351791/15a1ea49-85f3-4d3d-895e-e60a5a982cd3">

Have anyone had a similar issue before?

**Comment by romkatv** [comment]
`!w` in your prompt is an indicator that your current working directory is not writable.

Apple logo is `\uF179`.

**Reply by filipebcs** [reply]
Ok. Actually, the problem was solved by choosing Unicode when setting up Character Set. This prompt happens when you run the p10k configure as shown in the figure below

<img width="1030" alt="image" src="https://github.com/romkatv/powerlevel10k/assets/11351791/956dd669-9f97-4eb9-8339-0ae7efd80406">

I think my problem is solved. Now, I have to deal with Tabby not saving my history properly. Thank you for the help, @romkatv!
